### PR TITLE
draining info in delete server docs

### PR DESCRIPTION
**Pull request by manishtomar**
Add information about server being DRAINed before being deleted when using delete server API. @meker12 review please?

### CODE REVIEW

**Comment by manishtomar** [comment]
@meker12 I've made the changes. Could you kindly review it again and make any suggestions as inline comments?

File: api-docs/rst/dev-guide/concepts.rst [hunk 1]
```diff
@@ -639,10 +639,17 @@ and how many resources to delete.
...
 -  If new servers are in the process of being built and in a "pending"
-   state, these servers are deleted first.
+   state, these servers are chosen to be deleted first.
+
+-  After choosing the servers to delete autoscale will put the server's
+   corresponding load balancer's node in DRAINING **if** the group launch
+   configuration has cloud load balancer and ``draining_timeout`` in it.
+   The server will be deleted after ``draining_timeout`` has expired. If the
+   group launch configuration doesn't have cloud load balancer and ``draining_timeout``
+   then server is just deleted
```
Comment (meker12) [inline api-docs/rst/dev-guide/concepts.rst]: After selecting servers for deletion, the Autoscale process deletes each server immediately, unless the server has an associated load balancer that has been configured with a draining timeout period.  In these cases, Autoscale deletes the server after the draining timeout period for the load balancer expires. 

fter selecting the servers to delete, the Autoscale process puts any load balancers associated with the servers in DRAINING mode if the group launch configuration for the server includes the ``draining_timeout`` parameter. If the server A server is deleted after the draining timeout period expires. 
.

File: api-docs/rst/dev-guide/api-operations/methods/delete-delete-server-from-scaling-group-v1.0-tenantid-groups-groupid-servers-serverid.rst [hunk 2]
```diff
@@ -9,22 +9,29 @@ Delete server from scaling group
...
-   This operation can be useful if you have deleted a server by using a nova command or another method that Auto Scale does not recognize. In those cases, even though the server is deleted, Auto Scale calculates that the server still exists. You can use this operation to rectify this error and bring Auto Scale in sync with the previously unrecognized delete server operation.
+  - The *purge* parameter determines whether the server is removed from the account.
+    If the parameter is not specified, the value defaults to  ``purge=true``.
+    Specify ``purge=false`` to leave the server on the account.
+    This setting is useful if you want to investigate the server image after deleting it.
 
 .. note::
 
-   Deleting and replacing the server takes some time, how long depends mainly on the server image and complexity of the launch configuration settings of the replacement server.
+   Deleting and replacing the server takes some time. The time required depends on
```
Comment (meker12) [inline api-docs/rst/dev-guide/api-operations/methods/delete-delete-server-from-scaling-group-v1.0-tenantid-groups-groupid-servers-serverid.rst]: Deleting and replacing servers in a scaling group takes some time.

File: api-docs/rst/dev-guide/concepts.rst (hunk 1 above)
Comment (manishtomar) [inline api-docs/rst/dev-guide/concepts.rst]: > In these cases, Autoscale deletes the server after the draining timeout period for the load balancer expires.

This should probably be "In these cases, Autoscale puts the server's corresponding load balancer node in DRAINING for `draining_timeout` period after which is the server is deleted". Or some better text that mentions the CLB node being put in DRAINING before deleting the server.

**Comment by meker12** [comment]
@manishtomar  A few minor comments.  Back to you.

File: api-docs/rst/dev-guide/concepts.rst (hunk 1 above)
Comment (manishtomar) [inline api-docs/rst/dev-guide/concepts.rst]: @meker12 Could you respond to above comment?

Comment (meker12) [inline api-docs/rst/dev-guide/concepts.rst]: I updated the last sentence in the previous comment above.

Comment (manishtomar) [inline api-docs/rst/dev-guide/concepts.rst]: Sounds good. Thanks :)

**Pull request merged**